src/v2/api/controllers/AuthController.ts

```diff
@@ -160,13 +176,30 @@ export class AuthController {
     return this.userService.getUser(req.user.id);
   }
 
+  @ApiOkResponse()
```

Comment:
You must describe the request body of this endpoint. Go to ForgotPasswordDTO to do this.

Reply:
Nothing is returned. Its ok